### PR TITLE
Adding an image of the result, suggested change to code sample link

### PR DESCRIPTION
**Pull request by 1RedOne**
## Summary

Again, you can't give a GUI walkthrough without showing the end result!  🎃🦊

I uploaded an image of what this sample will generate and embedded it in the readme.  I also found that it looks like the snippet formatting was off, which leaves out most of the window declaration in XAML.  For example, it *should* post the [full snippet here](https://github.com/dotnet/samples/blob/master/snippets/csharp/VS_Snippets_Wpf/gridGetSetMethods/CSharp/Window1.xaml).  

But instead, an anchor link would cause most of the beginning of the snippet to be removed, this would lead to code which would not work when copied and pasted, and additionally, no link to the source itself!!

Heres what the anchor link generated:
```` xaml
<StackPanel Grid.Column="0" Grid.Row="0" HorizontalAlignment="Left" Orientation="Vertical">
            <Button Click="setCol0">Move Rectangle to Column 0</Button>
            <Button Click="setCol1">Move Rectangle to Column 1</Button>
            <Button Click="setCol2" Margin="0,0,0,10">Move Rectangle to Column 2</Button>
            <Button Click="setRow0">Move Rectangle to Row 0</Button>
            <Button Click="setRow1">Move Rectangle to Row 1</Button>
            <Button Click="setRow2" Margin="0,0,0,10">Move Rectangle to Row 2</Button>
            <Button Click="setColspan">Span All Columns</Button>
            <Button Click="setRowspan">Span All Rows</Button>
            <Button Click="clearAll">Clear All</Button>
          </StackPanel>
        </Grid>
      <Grid DockPanel.Dock="Top" Margin="0,10,15,0" HorizontalAlignment="Left" Name="grid1" ShowGridLines="True" Width="400" Height="400" Background="LightSteelBlue">
        <Grid.ColumnDefinitions>
        <ColumnDefinition/>
        <ColumnDefinition/>
        <ColumnDefinition/>
      </Grid.ColumnDefinitions>
      <Grid.RowDefinitions>
        <RowDefinition/>
        <RowDefinition/>
        <RowDefinition/>
      </Grid.RowDefinitions>
        
        <Rectangle Name="rect1" Fill="Silver" Grid.Column="0" Grid.Row="0"/>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="0" Grid.Row="0" Margin="5">Column 0, Row 0</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="1" Grid.Row="0" Margin="5">Column 1, Row 0</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="2" Grid.Row="0" Margin="5">Column 2, Row 0</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="0" Grid.Row="1" Margin="5">Column 0, Row 1</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="1" Grid.Row="1" Margin="5">Column 1, Row 1</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="2" Grid.Row="1" Margin="5">Column 2, Row 1</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="0" Grid.Row="2" Margin="5">Column 0, Row 2</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="1" Grid.Row="2" Margin="5">Column 1, Row 2</TextBlock>
        <TextBlock FontSize="15" HorizontalAlignment="Right" VerticalAlignment="Bottom" Grid.Column="2" Grid.Row="2" Margin="5">Column 2, Row 2</TextBlock>
      </Grid>
````
Note that with no Window, Dock or other starting declarations, this would cause a number of confusing code errors for clueless newbers like me.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by Thraka — DISMISSED** [review]

**Comment by mairaw** [comment]
Thanks @1RedOne! Also take a look at the changes I made with PR #8586 (lowercase filename, use dash to separate words, create a folder under media with the topic name, etc.). The path used by the migrated content from MSDN was not accurate, but we should try to stick to the guidelines when adding new ones.

**Comment by 1RedOne** [comment]
I renamed the file as well, per @mairaw's feedback. @Thraka, I can't see your new review feedback, however :(

File: docs/framework/wpf/controls/how-to-position-the-child-elements-of-a-grid.md [hunk 1]
```diff
@@ -14,12 +14,15 @@ This example shows how to use the get and set methods that are defined on <xref:
...
- [!code-xaml[gridGetSetMethods#1](../../../../samples/snippets/csharp/VS_Snippets_Wpf/gridGetSetMethods/CSharp/Window1.xaml#1)]  
+ [!code-xaml[gridGetSetMethods](../../../../samples/snippets/csharp/VS_Snippets_Wpf/gridGetSetMethods/CSharp/Window1.xaml)]  
   
  The following code-behind example handles the methods that the button <xref:System.Windows.Controls.Primitives.ButtonBase.Click> events raise. The example writes these method calls to <xref:System.Windows.Controls.TextBlock> elements that use related get methods to output the new property values as strings.  
   
  [!code-csharp[gridGetSetMethods#2](../../../../samples/snippets/csharp/VS_Snippets_Wpf/gridGetSetMethods/CSharp/Window1.xaml.cs#2)]
  [!code-vb[gridGetSetMethods#2](../../../../samples/snippets/visualbasic/VS_Snippets_Wpf/gridGetSetMethods/VisualBasic/Window1.xaml.vb#2)]  
+ Here is the finished result!
+ 
+ ![a screenshot depicts a WPF user interface with two columns, the right side has a 3 x 3 grid and the left has buttons to move a colored rectangle between the columns and rows of the grid](./media/gridmethodssample.png)
```
Comment (Thraka) [inline docs/framework/wpf/controls/how-to-position-the-child-elements-of-a-grid.md]: That looks much better. If you could, separate the words in the image file with -

`grid-method-sample.png`

**Review by Thraka — APPROVED** [review]

File: docs/framework/wpf/controls/how-to-position-the-child-elements-of-a-grid.md (hunk 1 above)
Comment (1RedOne) [reply docs/framework/wpf/controls/how-to-position-the-child-elements-of-a-grid.md]: done :)

Comment (Thraka) [reply docs/framework/wpf/controls/how-to-position-the-child-elements-of-a-grid.md]: @1RedOne yes, the file name here has to change too :) `(./media/gridmethodssample.png)`

**Comment by Thraka** [comment]
@1RedOne Awesome! Thanks a bunch!

**Pull request merged**